hibike/lib/Servo/README.adoc

```diff
@@ -0,0 +1,25 @@
+= Servo Library for Arduino =
```

Comment:
It actually doesn't.  I think it's a holdover from the same commit that fiddled with the make file.